src/locale/supported-content-lang-id.ts

```diff
@@ -9,4 +9,5 @@ export type SuppportedContentLangId =
   | "id-id"
   | "ja-jp"
   | "zh-cn"
-  | "zh-tw";
+  | "zh-tw"
```

Comment:
This file should be kept unchanged (it's the language code that _the anime game server_ supported)